src/components/users/index.js

```diff
@@ -235,6 +316,76 @@ class Users extends React.Component {
                         }
                       </BootstrapModal.Footer>
                     </BootstrapModal>;
+
+                  case 'inviteUser':
+                    return <BootstrapModal className='create-key-pair-modal' show={this.state.modal.visible} onHide={this.closeModal.bind(this)}>
+                      <BootstrapModal.Header closeButton>
+                        <BootstrapModal.Title>Invite a New User</BootstrapModal.Title>
+                      </BootstrapModal.Header>
+
+                      <BootstrapModal.Body>
+                        <form>
+                        <p>Creating an invitation is the way to get new people onto this installation.</p>
+                        <p>You can chose whether or not to send the invitee an email, and which organization the invitee should become a member of once they accept their invitation.</p>
+                        <p>Invitations are valid for 48 hours, after which you&apos;ll have to re-invite the person.</p>
```

Comment:
> after which you&apos;ll have to re-invite the person

I think this can be guessed. Also not relevant in the situation the user is right now.